lib/builder-webpack4/package.json

```diff
@@ -111,7 +111,7 @@
   },
   "devDependencies": {
     "@types/case-sensitive-paths-webpack-plugin": "^2.1.4",
-    "@types/terser-webpack-plugin": "^5.0.2",
+    "@types/terser-webpack-plugin": "^4.2.0",
```

Comment:
The v5 of this is for the v5 version of the package, which is for webpack 5, using the wrong version will cause typings conflicts between webpack 4 and 5